### ISSUE TITLE
chore: add `සිංහල - si-LK` language support

### ISSUE THREAD

**Issue by brionmario** [asgardeo/javascript#155]
### Current Limitation

Currently, the Asgardeo JavaScript SDK does not have support for **Sinhala (සිංහල - si-LK)**. Users who prefer Sinhala will not see UI components translated, limiting accessibility and localization for Sri Lankan users.

> [!Important]
> Contributors for this task should be **native Sinhala speakers or have professional fluency** to ensure accurate, culturally appropriate translations.

### Suggested Improvement

Add **Sinhala (`si-LK`) language support** to the `@asgardeo/i18n` package.

**Implementation Plan:**

1. Create a new locale file:

    Follow https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package
    
3. Add documentation for usage:  
   - Update the SDK docs: https://wso2.com/asgardeo/docs/sdks/react/overview/

### Please select the package issue is related to

@asgardeo/i18n

### Version

current

### Reporter Checklist

- [x] I have searched the existing issues and this is not a duplicate.
- [x] I have provided all the necessary information.
- [x] I have verified the improvement is not available in the latest version of the package.

**Comment by brionmario** [comment]
> Hey [@brionmario](https://github.com/brionmario) Checked the guidelines 🙌 Will reach out if any questions.

Great.
There's an update to the contribution guide. Incase you missed, please have a look again https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package

Cheers!

**Comment by sagara-gunathunga** [comment]
@TheekshanaCN Congratulations on completing this issue! Unless you’ve already done so, please complete the form for this issue as mentioned on the WSO2 Hacktoberfest webpage: https://wso2.com/hacktoberfest/

**Comment by TheekshanaCN** [comment]
@sagara-gunathunga Thank you! 😊 I’ve submitted the form .. happy to be part of WSO2 Hacktoberfest!
